packages/core/src/components/Dialog/__snapshots__/Dialog.test.jsx.snap

```diff
@@ -149,7 +149,7 @@ exports[`Dialog renders react-aria-modal 1`] = `
           active={true}
           focusTrapOptions={
             Object {
-              "initialFocus": undefined,
+              "escapeDeactivates": true,
```

Comment:
For whatever reason this snapshot changes with an upgraded node and it was bothering me for a while.